### ISSUE TITLE
Idea: Use available space on larger screens and improve content structure.

### ISSUE THREAD

**Issue by Gong-Bao-Chicken** [getkirby-v2/panel#917]
**TLDR:** Here is a sketch that shows, that there is plenty of space for a second navigational or settings column on the left and maybe an anchor navigation on the top of the content view or in the new / additional column. Which would improve usability in the panel on larger screens... 

![improvements to kirby panel](https://cloud.githubusercontent.com/assets/3042610/18609146/2118889c-7cfb-11e6-8b20-e19f16f5c90f.jpg)

(Captured the Screenshot at 2560x1920, but even after inserting an extra column there was still plenty of space for content...)

**First**: Because larger screens are more popular these days (retina displays, and 1920+ px horizontally), it would be cool if kirby panel would use this extra space to improve it's page navigation.
- Maybe Split the current navigation on the left, into a tree menu with copy / move capability on the left, and page settings on the right.
- Or parent pages on the left - child pages with settings and files on the right? 
  ... other ideas? 

**Second**: With 16:9 displays [ becoming more popular](https://en.wikipedia.org/wiki/Display_aspect_ratio) than displays with aspect ratio of 19:10 and 4:3 vertical space is limited. Therefore my suggest is to add a tabbed or anchored navigation for pages that take a lot of vertical space, either in a new column on large screens or at the top of the content pages (this is available as plugin, but would be even better in core).
